index.rst

```diff
@@ -6,24 +6,474 @@
+  deepCoadd_wcs = butler.get('deepCoadd_calexp.wcs', dataId=dataId)
+  deepCoadd_WCSfMd = WCS(deepCoadd_wcs.getFitsMetadata())
+  deepCoadd_extent = (deepCoadd_bbox.beginX, deepCoadd_bbox.endX, deepCoadd_bbox.beginY, deepCoadd_bbox.endY)
+  plt.subplot(projection=deepCoadd_WCSfMd)
+  plt.imshow(deepCoadd.image.array, cmap='gray', vmin=0, vmax=2, extent=deepCoadd_extent, origin='lower')
+
+
+Clearing memory
+---------------
+
+These are optional methods for keeping memory use manageable in notebooks which are, e.g., demonstrating
+data visualization techniques with big datasets.
+
+To reduce the memory footprint of a notebook, remove figures once they're no longer needed.
+See the ``remove_figure`` function defined in the DP0 notebook `03_Image_Display_and_Manipulation.ipynb <https://github.com/rubin-dp0/tutorial-notebooks/blob/main/03a_Image_Display_and_Manipulation.ipynb>`__.
+
+*A method for clearing memory of, for example, large arrays that are not going to be used further on in the notebook is in development.*
+
+
+Assert statements
+-----------------
+
+Where essential, or where a very specific value is expected, the ``assert`` command can be used to demonstrate to users that a condition is true.
+
+For example, ``assert`` statements can be used to confirm that service objects like TAP are not ``None`` or ``null`` before moving on and using that instance,
+or to check that values meet expectations (e.g., total rows returned from a query).
+
+However, take care not to use ``assert`` statements when, e.g., querying dynamic (prompt) datasets.
+Consider more pedagogical alternatives when possible (e.g., printing schema columns would also fail if the TAP service was not instantiated).
+
+
+Known warnings
+--------------
+
+If a code cell produces a warning which is known, the preferred method to ignore warnings is to add a markdown cell,
+*before* the code cell which produces the warning, to tell the user it is OK to ignore.
+
+*The appropriate use-cases for ignoring categories of warnings (below) is still under consideration.*
+This is not preferred because ignore categories of warnings can allow real issues to go unnoticed.
+
+::
+
+  warnings.simplefilter("ignore", category=UserWarning)
+
+
+Markdown style
+--------------
+
+Any references to variables used in code cells or any code commands should be in ``monospaced font``.
+
+Use of indented text should be limited to warnings and notices, e.g., ``> **Warning:** the following cell...``. 
+
+*The most appropriate narrative voice for tutorials is still under consideration.*
+Adopt your own style for now.
+
+
+Code cell comments
+------------------
+
+Avoid using comments within a code cell as documentation.
+Markdown cells are the preferred way to provide descriptive text.
+
+
+Code cell style standard PEP8
+-----------------------------
+
+``PEP8`` is the standard, and ``flake8`` is a tool to ensure compliance with the standards.
```

Comment:
Perhaps place a link with a reference to what PEP8 is?

Reply:
Changed the name of the link text